type-definitions/most.d.ts

```diff
@@ -90,11 +93,14 @@ declare interface DisposeFn {
   (): void|Promise<any>;
 }
 
-export function create<A>(f: (add: (a:A) => any, end: (x:any) => any, error: (e:Error) => any) => void|DisposeFn): Stream<A>;
+export function create<A>(f: (add: (a: A) => any, end: (x: any) => any, error: (e: Error) => any) => void|DisposeFn): Stream<A>;
 export function just<A>(a: A): Stream<A>;
 export function of<A>(a: A): Stream<A>;
+export function empty(): Stream<any>;
+export function never(): Stream<any>;
 export function from<A>(as: Iterable<A>): Stream<A>;
 export function periodic<A>(period: number, a?: A): Stream<A>;
+export function fromEvent(event: string, target: any, useCapture?: boolean): Stream<Event>;
```

Comment:
Ah, this one is tricky!  Is it possible to provide two signatures for fromEvent?  i.e. since it handles both EventTarget and EventEmitter, we may be able to gain a small amount of type safety if TS supports polymorphic functions ... something like:

```js
export function fromEvent(event: string, target: EventTarget, useCapture?: boolean): Stream<Event>;
export function fromEvent(event: string, target: EventEmitter): Stream<any>;
```

I can't see a way to avoid the `any` in the EventEmitter version's return type :(

Anyway, if that's not possible, then maybe we can use a union type for `target`, e.g. `EventTarget | EventEmitter`.

If that still won't work because TS doesn't know about EventTarget and EventEmitter, then maybe we can define structural types for them?  i.e. at least define what methods they must have?